### PR TITLE
Removes moved translator instruction

### PR DESCRIPTION
**Pull request by matsduf**
zonemaster/zonemaster#1015 moves the translator instruction to the main Zonemaster repository, or technically add such a document there. When that PR has been merged, the PR should be merged to remove it from this repository.

* Remove the instruction for translators. Has been created in zonemaster/zonemaster instead.
* Replace moved document with a document with referral.
* Update link in the general translation document.

Please note that the link in the document does not work until zonemaster/zonemaster#1015 has been merged.

## How to test this PR

Documentation only.

### CODE REVIEW

File: docs/Translation-translators.md [hunk 1]
```diff
@@ -1,327 +1,8 @@
-# Instructions for translators
+This document has been moved to the main Zonemaster repository.
 
-See the general [Translation] document for overall documentation on
-translation.
+Go to "[Instructions for translators]".
```
Comment (mattias-p) [inline docs/Translation-translators.md:3]: Maybe just like this?
```
This document has moved [here][Instructions for translators].
```

**Review by mattias-p — DISMISSED** [review]

File: docs/Translation-translators.md (hunk 1 above)
Comment (matsduf) [reply docs/Translation-translators.md]: Done, but I do not see why that should be better?

Comment (mattias-p) [reply docs/Translation-translators.md]: It's shorter and more straight to the point.

**Review by mattias-p — APPROVED** [review]

File: docs/Translation-translators.md (hunk 1 above)
Comment (matsduf) [reply docs/Translation-translators.md]: There were two lines and one link. It would work fine.

**Review by blacksponge — APPROVED** [review]

**Pull request merged**